### ISSUE TITLE
Not able to run any module

### ISSUE THREAD

**Issue by 4ndersson** [netbox-community/ansible_modules#458]
##### ISSUE TYPE

 - Bug Report

###### Ansible:
2.9.6

###### Netbox:
2.7.11

###### Python:
3.8.5

###### Collection:
2.1.0

##### SUMMARY
Not able to run any module.

##### STEPS TO REPRODUCE

```yaml
- hosts: netbox
  gather_facts: no
  connection: local

  tasks:
    - name: get networks from netbox
      netbox.netbox.netbox_prefix:
        netbox_url: "http://{{ inventory_hostname }}"
        netbox_token: "{{ token }}"
        data:
          parent: 10.160.0.0/20
          prefix_length: 10.160.0.0/24
        state: present
        first_available: yes
```

##### EXPECTED RESULTS
Playbook run successfully.

##### ACTUAL RESULTS

```
TASK [get networks from netbox] *********************************************************************
task path: /root/infra/ansible/labb.yml:6
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: root
<localhost> EXEC /bin/sh -c 'echo ~root && sleep 0'
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /root/.ansible/tmp/ansible-tmp-1614877158.8355212-211335509262300 `" && echo ansible-tmp-1614877158.8355212-211335509262300="` echo /root/.ansible/tmp/ansible-tmp-1614877158.8355212-211335509262300 `" ) && sleep 0'
<localhost> EXEC /bin/sh -c 'rm -f -r /root/.ansible/tmp/ansible-tmp-1614877158.8355212-211335509262300/ > /dev/null 2>&1 && sleep 0'
The full traceback is:
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/ansible/executor/task_executor.py", line 146, in run
    res = self._execute()
  File "/usr/lib/python3/dist-packages/ansible/executor/task_executor.py", line 645, in _execute
    result = self._handler.run(task_vars=variables)
  File "/usr/lib/python3/dist-packages/ansible/plugins/action/normal.py", line 46, in run
    result = merge_hash(result, self._execute_module(task_vars=task_vars, wrap_async=wrap_async))
  File "/usr/lib/python3/dist-packages/ansible/plugins/action/__init__.py", line 815, in _execute_module
    (module_style, shebang, module_data, module_path) = self._configure_module(module_name=module_name, module_args=module_args, task_vars=task_vars)
  File "/usr/lib/python3/dist-packages/ansible/plugins/action/__init__.py", line 204, in _configure_module
    (module_data, module_style, module_shebang) = modify_module(module_name, module_path, module_args, self._templar,
  File "/usr/lib/python3/dist-packages/ansible/executor/module_common.py", line 1255, in modify_module
    (b_module_data, module_style, shebang) = _find_module_utils(module_name, b_module_data, module_path, module_args, task_vars, templar, module_compression,
  File "/usr/lib/python3/dist-packages/ansible/executor/module_common.py", line 1094, in _find_module_utils
    recursive_finder(module_name, remote_module_fqn, b_module_data, py_module_names,
  File "/usr/lib/python3/dist-packages/ansible/executor/module_common.py", line 730, in recursive_finder
    module_info = CollectionModuleInfo(py_module_name[-idx],
  File "/usr/lib/python3/dist-packages/ansible/executor/module_common.py", line 650, in __init__
    self.get_source()
  File "/usr/lib/python3/dist-packages/ansible/executor/module_common.py", line 666, in get_source
    data = pkgutil.get_data(to_native(self._package_name), to_native(self._mod_name + '.py'))
  File "/usr/lib/python3.8/pkgutil.py", line 619, in get_data
    spec = importlib.util.find_spec(package)
  File "/usr/lib/python3.8/importlib/util.py", line 114, in find_spec
    raise ValueError('{}.__spec__ is None'.format(name))
ValueError: ansible_collections.netbox.netbox.plugins.module_utils.netbox_ipam.__spec__ is None
fatal: [nb.miradot.lab]: FAILED! => {
    "msg": "Unexpected failure during module execution.",
    "stdout": ""
}

PLAY RECAP ******************************************************************************************
nb.miradot.lab             : ok=0    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0
```

**Comment by lassizci** [comment]
I just noticed the same issue, although with old 0.2.1.  `ansible.netcommon` version 2.0.0 seemed to break this. The dependency doesn't seem to be there in `master` anymore though.

**Comment by lassizci** [comment]
https://github.com/netbox-community/ansible_modules/issues/452

**Comment by 4ndersson** [comment]
@lassizci Great stuff, when using `ansible.netcommon` version 1.5.0 instead it seems to work!

**Comment by FragmentedPacket** [comment]
Released v3.0.0 to remove all other dependencies and issue should be resolved.

Sorry about that!
